cocotb/share/makefiles/simulators/Makefile.activehdl

```diff
@@ -0,0 +1,74 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Common Makefile for Aldec Active-HDL simulator
+
+CMD_BIN := vsimsa
+
+ifdef ACTIVEHDL_BIN_DIR
+    CMD := $(shell :; command -v $(ACTIVEHDL_BIN_DIR)/$(CMD_BIN) 2>/dev/null)
+else
+    # auto-detect bin dir from system path
+    CMD := $(shell :; command -v $(CMD_BIN) 2>/dev/null)
+endif
+
+ALOG_ARGS += -timescale $(COCOTB_HDL_TIMEUNIT)/$(COCOTB_HDL_TIMEPRECISION)
+
+# below allows for maintaining legacy syntax as well as enables using cross-simulator vars COMPILE_ARGS/SIM_ARGS
+ALOG_ARGS += $(COMPILE_ARGS)
+ACOM_ARGS += $(COMPILE_ARGS)
+ASIM_ARGS += $(SIM_ARGS)
+
+RTL_LIBRARY ?= work
+ALOG_ARGS += +define+COCOTB_SIM -dbg
+ACOM_ARGS += -dbg
+
+GPI_EXTRA:=
+ifeq ($(TOPLEVEL_LANG),verilog)
+    GPI_ARGS = -pli \"$(call to_tcl_path,$(LIB_DIR)/libcocotbvpi_aldec)\"
+ifneq ($(VHDL_SOURCES),)
+    GPI_EXTRA = cocotbvhpi_aldec:cocotbvhpi_entry_point
+endif
+
+else ifeq ($(TOPLEVEL_LANG),vhdl)
+    GPI_ARGS = -loadvhpi \"$(call to_tcl_path,$(LIB_DIR)/libcocotbvhpi_aldec):vhpi_startup_routines_bootstrap\"
```

Comment:
```suggestion
    # backslashes needed because we embed in `echo` below
    GPI_ARGS = -loadvhpi \"$(call to_tcl_path,$(LIB_DIR)/libcocotbvhpi_aldec):vhpi_startup_routines_bootstrap\"
```